src/hydranfc/hydranfc_cmd_sniff.c

```diff
@@ -1022,12 +1026,12 @@ void hydranfc_sniff_14443A(t_hydra_console *con, bool start_of_frame, bool end_o
 
 			nb_cycles_end = get_cyclecounter();
 
-			if (sniff_pcap_output) {
-				// TODO RSSI can be useful for ASCII logs as well
-				data_buf[0] = RSSI_LEVELS;
-				Trf797xReadSingle(data_buf, 1);
-				pow = data_buf[0];
-			}
+			//	removed as it's too slow for sniffing
```

Comment:
Remove all comment lines related to RSSI for cleaner code and remove data_buf[1]; to avoid a warning